packages/main/src/themes/GridHeaderCell.css

```diff
@@ -1,6 +1,7 @@
 :host {
     font-family: var(--sapFontSemiboldDuplexFamily);
     color: var(--sapList_HeaderTextColor);
+    justify-content: var(--_ui5-grid-cell-base-hAlign);

```

Comment:
Is there a specific reason, why GridCell has `justify-content: var(--_ui5-grid-cell-base-hAlign, left);`, while GridHeaderCell has it without `left`?
Couldnt' this be put into GridCellBase, if we got nearly the same CSS for both cell types?